### ISSUE TITLE
Investigate hyperparameter combination

### ISSUE THREAD

**Issue by JWL16038** [JWL16038/classifind#11]
This task will investigate the combination of hyper-parameters that could be used to train the first iteration of the deep learning model. Some components that can be used for this model are:
- Optimiser: Adam
- Loss (objective) function: Forward Cross Entropy
- Gradient clipping (??)
